cmd/hamctl/command/root.go

```diff
@@ -54,14 +53,10 @@ func NewRoot(version *string) (*cobra.Command, error) {
 			if service == "" {
 				missingFlags = append(missingFlags, "service")
 			}
-			if email == "" {
-				committer, err := gitConfigAPI.CommitterDetails()
-				if err != nil {
-					missingFlags = append(missingFlags, "user-email")
-				}
-				email = committer.Email
+			if err := setCallerEmailFromCommitter(gitConfigAPI, &client); err != nil {
```

Comment:
This doesn't respect setting the `user-email` flag directly, does it?
In that case `email` will be set up front?

Specifically this is a change as the `if email == ""` is removed?

Reply:
I've included the args as well